docs/Validation-and-Serialization.md

```diff
@@ -599,14 +599,18 @@ fastify.setErrorHandler(function (error, request, reply) {
 ```
 
 If you want custom error response in schema without headaches and quickly, you can take a look at [`ajv-errors`](https://github.com/epoberezkin/ajv-errors). Check out the [example](https://github.com/fastify/example/blob/HEAD/validation-messages/custom-errors-messages.js) usage.
+> Make sure to install version 1.0.1 of `ajv-errors`, because later versions of it are not compatible with AJV v6.
 
 Below is an example showing how to add **custom error messages for each property** of a schema by supplying custom AJV options.
 Inline comments in the schema below describe how to configure it to show a different error message for each case:
 
 ```js
 const fastify = Fastify({
   ajv: {
-    customOptions: { jsonPointers: true },
+    customOptions: {
+      jsonPointers: true,
+      allErrors: true
+    },
```

Comment:
Oof. We need to update this example with a warning that implementing it opens the service up to a denial of service attack.

Reply:
Should I just add the warning as a comment in front of the option:
```js
customOptions: {
  jsonPointers: true,
  allErrors: true // Warning: Enabling this option may open the service up to a denial of service attack
},
```

Reply:
I would add a note box with the link to the CVE
https://www.cvedetails.com/cve/CVE-2020-8192/